### DISCUSSION TITLE
How to do Fast tDCS Simulation with Personalized T1 file

### DISCUSSION THREAD

**Discussion by JoonNeuro** [simnibs/simnibs#249] [7.Creating head models]
Dear SimNIBS team and experts,

When I preceeded `charm` code. It took about an hour to create an m2m folder using and run a simulation using it.

Currently, the only file I have is the Personalized T1 file.

I would like to ask if there is a way to make a faster tDCS simulation using it. 
Is there downsample option in `charm`?
Or would it be faster to just simulate or optimize the matrix without creating a mesh, then how?

Thanks, 
Joonho Lee

**Comment by TorgeW** [comment]
While running charm to create a head model can take some time, this process has to be done only once per subject. 
After this, running one or more tDCS simulations should be rather fast. 

For tDCS we do not recommend any down sampling because a high definition head model is required for accurate results.

**Reply by JoonNeuro** [reply]
Thank you for reply!
